test/views/network_status_view_test.js

```diff
@@ -4,24 +4,24 @@ describe('NetworkStatusView', function() {
         var networkStatusView;
         var socketStatus = WebSocket.OPEN;
 
-        var oldGetMessage;
+        var oldI18n;
         var oldGetSocketStatus;
 
         /* BEGIN stubbing globals */
         before(function() {
             oldGetSocketStatus = window.getSocketStatus;
-            /* chrome i18n support is missing in 'regular' webpages */
-            window.chrome = window.chrome || {};
-            window.chrome.i18n = { getMessage: function(message, args) {
-                // translationMessageName-arg1-arg2
+            oldI18n = window.i18n;
+
+            // translationMessageName-arg1-arg2
+            window.i18n = function(message, args) {
```

Comment:
Looks like tests in here depends on this broken i18n stub because we never had proper i18n in tests before. File a chore for later to kill this stub?